app/src/main/res/drawable/ic_smartphone_20dp.xml

```diff
@@ -0,0 +1,4 @@
+<vector android:height="20dp" android:viewportHeight="35.0"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_smartphone_20dp` appears to be unused
 Unused resources make applications larger and slow down builds.